### PR TITLE
[60414] Introduce the restart option

### PR DESCRIPTION
**Pull request by akashchi**

### CODE REVIEW

File: starting_package/openvino_workbench/container.py [hunk 1]
```diff
@@ -22,11 +22,12 @@
 import sys
 import time
 
-from openvino_workbench.constants import DL_WB_LOGO, PRE_STAGE_MESSAGES, STAGE_COMPLETE_MESSAGES, INTERNAL_PORT
```
Comment (akashchi) [inline starting_package/openvino_workbench/container.py:25]: Used workbench's `.pylintrc` file and aligned ordering here and in other files.

**Review by tsavina — APPROVED** [review]

File: starting_package/openvino_workbench/arguments_parser.py [hunk 2]
```diff
@@ -19,13 +19,33 @@
...
 
 
 def get_proxy_from_env(proxy: str) -> Optional[str]:
     return os.getenv(proxy) or os.getenv(proxy.upper())
 
 
+def validate_args_for_restart(args: argparse.Namespace, parser: argparse.ArgumentParser):
+    # If detached restart is needed then there should be exactly 4 arguments
+    # and '--detached' should be one of them
+    if args.detached and len(sys.argv) != 4:
```
Comment (artyomtugaryov) [inline starting_package/openvino_workbench/arguments_parser.py:33]: I think, we need to use `args ` to check

File: starting_package/openvino_workbench/container.py [hunk 3]
```diff
@@ -52,27 +53,25 @@ def wait_for_stage_to_complete(pre_message: str,
...
-    print(f'{message}?token={tokens["url_token"]}\n'
-          '\nNote: Authentication with the token inside this link is available only once.\n'
-          f'The link expires after you click it. Use your login token at {link} to '
-          'authenticate again.\n')
+    # Show finishing message from the container logs
+    container = docker_client.containers.get(container_name)
+    all_logs = container.logs().decode('utf-8').rstrip()
+    finishing_message_start = all_logs.rfind(WORKBENCH_READY_MESSAGE)
+    print(f'\n{all_logs[finishing_message_start:]}')
+    del all_logs
```
Comment (artyomtugaryov) [inline starting_package/openvino_workbench/container.py:64]: ?

File: starting_package/openvino_workbench/arguments_parser.py (hunk 2 above)
Comment (akashchi) [reply starting_package/openvino_workbench/arguments_parser.py:33]: Unfortunately, `argparse.Namespace` has nothing related to its length / how much arguments are passed.

File: starting_package/openvino_workbench/container.py (hunk 3 above)
Comment (akashchi) [reply starting_package/openvino_workbench/container.py:64]: If we restart a container, we know nothing about the ip, port, protocol, etc. so we cannot construct a proper finishing message. On the other hand, there is a finishing message in container logs so I've decided to simplify the finishing message logic and rely on the container logs both for the first start of a container and for the restart.

**Review by artyomtugaryov — APPROVED** [review]

File: starting_package/openvino_workbench/container.py (hunk 3 above)
Comment (demid5111) [reply starting_package/openvino_workbench/container.py:64]: omg, that is strange

**Pull request merged**